internal/pkg/cli/git.go

```diff
@@ -5,18 +5,43 @@ package cli
 
 import (
 	"bytes"
+	"fmt"
 	"strings"
 
 	"github.com/aws/copilot-cli/internal/pkg/term/command"
+
+	"github.com/aws/copilot-cli/internal/pkg/term/log"
+
+	"github.com/aws/copilot-cli/internal/pkg/term/prompt"
 )
 
-func getVersionTag(runner runner) (string, error) {
-	var b bytes.Buffer
+const (
+	inputImageTagPrompt = "Input an image tag value:"
+)
 
-	if err := runner.Run("git", []string{"describe", "--always"}, command.Stdout(&b)); err != nil {
+func getVersionTag(runner runner) (string, error) {
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+	if err := runner.Run("git", []string{"describe", "--always"}, command.Stdout(&stdout), command.Stderr(&stderr)); err != nil {
 		return "", err
 	}
 
 	// NOTE: `git describe` output bytes includes a `\n` character, so we trim it out.
-	return strings.TrimSpace(b.String()), nil
+	return strings.TrimSpace(stdout.String()), nil
+}
+
+func askImageTag(tag string, prompter prompter, cmd runner) (string, error) {
+	var err error
+	if tag == "" {
```

Comment:
nit: can we flip the if-statement to reduce nesting?
```go
if tag != "" {
   return tag, nil
}
// the rest
```

Reply:
Oh, good call-- thanks. I also took out `var err error`, which is no longer necessary.